examples/visual-tests/amp-video-docking/amp-video-docking.js

```diff
@@ -33,57 +33,80 @@ async function dock(page, name) {
   await verifySelectorsVisible(page, name, ['.amp-video-docked-shadow']);
 }
 
-async function hoverDockArea(page, name) {
-  await page.hover('.i-amphtml-video-docked-overlay');
+async function activateControlsBy(page, name, tapOrHover) {
+  await page[tapOrHover].call(page, '.i-amphtml-video-docked-overlay');
```

Comment:
While this is a test and not compiled / typed, I think we should still write this as if it were. That is, something like:

```javascript
if (tapOrHover === 'hover') {
  await page.hover('.i-amphtml-video-docked-overlay');
} else {
  await page.tap('.i-amphtml-video-docked-overlay');
}
```